back/engines/commercial/admin_api/spec/acceptance/projects_spec.rb

```diff
@@ -46,24 +46,26 @@
       parameter :template_yaml, "The yml template for the project to import", required: true
     end
 
+    let(:tenant) { create(:test_tenant, host: 'abi-dhabu.citizenlab.co') }
+    let(:template) do
+      create(:test_tenant, host: 'abu-dhabi.citizenlab.co').switch do
+        project = create(:project_xl, phases_count: 3, images_count: 0, files_count: 0) # no images nor files because URL's will not be available
+        AdminApi::ProjectCopyService.new.export(project)
+      end
+    end
+
     example "Import a project template" do
-      tn1 = create(:test_tenant, host: 'abu-dhabi.citizenlab.co')
-      tn2 = create(:test_tenant, host: 'las-vegas.citizenlab.co')
+      do_request(tenant_id: tenant.id, project: { template_yaml: template.to_yaml })
 
-      template, project = nil
-      Apartment::Tenant.switch(tn1.schema_name) do
-        project = create(:project_xl, phases_count: 8, images_count: 0, files_count: 0) # no images nor files because URL's will not be available
-        template = AdminApi::ProjectCopyService.new.export(project)
-      end
+      expect(status).to eq(200)
+      expect(DumpTenantJob).to have_been_enqueued if defined?(NLP)
 
-      do_request(tenant_id: tn2.id, project: {template_yaml: template.to_yaml})
-      expect(status).to eq 200
-      Apartment::Tenant.switch(tn2.schema_name) do
-        project_copy = Project.first
+      tenant.switch do
+        project = Project.first
 
-        expect(template['models']['project'].first.dig('title_multiloc','en')).to eq project_copy.title_multiloc['en']
-        expect(template['models']['phase'].size).to eq project_copy.phases.count
-        expect(template['models']['phase'].map{|h| h['start_at']}).to match project_copy.phases.map(&:start_at).map(&:iso8601)
+        expect(template['models']['project'].first.dig('title_multiloc', 'en')).to eq project.title_multiloc['en']
+        expect(template['models']['phase'].size).to eq project.phases.count
+        expect(template['models']['phase'].pluck('start_at')).to match project.phases.map(&:start_at).map(&:iso8601)
```

Comment:
I know it was already written like this, but I think we should replace `match` by ` match_array` here. Most random test failures are caused by this.